.eslintrc.js

```diff
@@ -114,6 +116,9 @@ module.exports = {
     'no-empty': WARNING,
     'no-prototype-builtins': WARNING,
     'no-case-declarations': WARNING,
+    'no-undef': OFF,
```

Comment:
Disabled this 'no-undef' rule as this causes a bunch of errors and to prevent double parsing in TS according to the recommendation from the issue typescript-eslint/typescript-eslint#2508